requirements/base.txt

```diff
@@ -51,4 +51,4 @@ six==1.11.0               # via django-markup, djangorestframework-gis, drf-yasg
 unidecode==1.0.22         # via zds-schema
 uritemplate==3.0.0        # via coreapi, drf-yasg
 urllib3==1.24.1           # via requests
-zds-schema==0.20.6
+zds-schema==0.21.3
```

Comment:
There's a script `bin/compile_dependencies.sh` which will take care of all deps - I'm missing changes to `requirements/jenkins.txt` here :)

Reply:
Done